graphmuse/nn/conv/gat.py

```diff
@@ -161,7 +181,7 @@ class CustomGATConv(MessagePassing):
         {\sum_{k \in \mathcal{N}(i) \cup \{ i \}}
         \exp\left(\mathrm{LeakyReLU}\left(
         \mathbf{a}^{\top}_{s} \mathbf{\Theta}_{s}\mathbf{x}_i
-        + \mathbf{a}^{\top}_{t} \mathbf{\Theta}_{t}\mathbf{x}_k
+        + \mathbf{a}^{\top}_{t}\mathbf{\Theta}_{t}\mathbf{x}_k
```

Comment:
There is a missing space here for latex code I think

Reply:
not a problem.